### PR TITLE
#25168 Microsoft Calendar O365 README > Split out admin content to child pages

### PR DESCRIPTION
**Pull request by Reene-Simon**
#### Summary
This PR organizes the README section as per the instructions in the issue description. I have used the reviews given for a similar issue, namely the Bitbucket README and have tried to incorporate the instructions given by the reviewers. Hence, I haven't relinked the images as per given in the ticket. If this step is necessary, do let me know and I'll make the required changes. These are the main changes that have been done:

1. Created a table of contents as mentioned
2. Made a new admin-guide file under the docs folder 
3. Moved all configuration details into the admin-guide file
4. Linked all the links and the table of contents according to the changes made

#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/25168

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]
Thanks, @Reene-Simon!

File: docs/admin-guide.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Admin Guide
+
+- [Configuration](#configuration)
+
+## Configuration
+
+### Step 1: Create Mattermost App in Azure
```
Comment (mickmister) [inline docs/admin-guide.md:7]: Might as well include in the table of contents links to "Step 1" and "Step 2"

File: README.md [hunk 2]
```diff
@@ -10,93 +10,22 @@
...
-2. [Features](#features)
-3. [Configuration](#configuration)
+- [Overview](#overview)
+- [Features](#features)
+- [Admin Guide](#admin-guide)
 
 ## Overview
 
-This plugin supports a two-way integration between Mattermost and Microsoft Outlook Calendar. For a stable production release, please download the latest version from the Plugin Marketplace and follow [these instructions](#configuration) to install and configure the plugin.
+This plugin supports a two-way integration between Mattermost and Microsoft Outlook Calendar. For a stable production release, please download the latest version from the Plugin Marketplace and follow [these instructions](./docs/admin-guide.md#configuration) to install and configure the plugin.
```
Comment (mickmister) [inline README.md:21]: We actually recently removed this plugin from the marketplace. We can instead say something like this:

```suggestion
This plugin supports a two-way integration between Mattermost and Microsoft Outlook Calendar. For a stable production release, please download the latest version from the [releases page](https://github.com/mattermost/mattermost-plugin-mscalendar/releases) and follow [these instructions](./docs/admin-guide.md#configuration) to install and configure the plugin.
```

@fmartingr The "latest" release is a pre-release called `v1.0.0-rc1`. Should we keep this release around?

**Review by mickmister — COMMENTED** [review]
Thanks @Reene-Simon! Just a couple minor suggestions. Otherwise LGTM!

**Comment by Reene-Simon** [comment]
@cwarnermm @mickmister all the changes have been made. Please do review it.

**Review by mickmister — APPROVED** [review]
LGTM :tada:

**Review by mickmister — APPROVED** [review]

**Comment by mickmister** [comment]
@hanzei There is a similar lint issue here too https://github.com/mattermost/mattermost-plugin-bitbucket/pull/127#discussion_r1384577956

**Comment by mickmister** [comment]
@Reene-Simon We are in the process of resolving a linting issue in our CI pipeline. We can merge once this is fixed

**Comment by Reene-Simon** [comment]
@mickmister yeah sure no worries!

**Comment by mickmister** [comment]
Thanks @Reene-Simon!

**Pull request merged**